src/App.tsx

```diff
@@ -1,62 +1,120 @@
-import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import cn from 'classnames';
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+export const goodsFromServer: Good[] = [
+  { name: 'Dumplings' },
+  { name: 'Carrot' },
+  { name: 'Eggs' },
+  { name: 'Ice cream' },
+  { name: 'Apple' },
+  { name: 'Bread' },
+  { name: 'Fish' },
+  { name: 'Honey' },
+  { name: 'Jam' },
+  { name: 'Garlic' },
 ];
 
-export const App: React.FC = () => {
+interface Good {
+  name: string
+}
+
+export const App = () => {
+  const defaultValue: Good[] = [...goodsFromServer];
```

Comment:
No need in this, just do this `const [goods, setGoods] = useState<Good[]>(goodsFromServer);`